ember-toucan-core/package.json

```diff
@@ -113,7 +113,12 @@
     "type": "addon",
     "main": "addon-main.cjs",
     "app-js": {
-      "./components/button.js": "./dist/_app_/components/button.js"
+      "./components/button.js": "./dist/_app_/components/button.js",
+      "./components/form/control.js": "./dist/_app_/components/form/control.js",
+      "./components/form/error.js": "./dist/_app_/components/form/error.js",
+      "./components/form/field.js": "./dist/_app_/components/form/field.js",
+      "./components/form/hint.js": "./dist/_app_/components/form/hint.js",
+      "./components/form/label.js": "./dist/_app_/components/form/label.js"
```

Comment:
This is now making also the yielded components available for direct usage, as in `<Form::Label>`. Do we want this? We could also try to hide them as much as possible from users, and only make them available yielded by `<Form::Field>`. This is what headless is doing.

To do this, we would need to make tighten the glob(s) for appReexports  in the rollup config.

Reply:
Perfect, thank you for the suggestion!  Went ahead and updated via https://github.com/CrowdStrike/ember-toucan-core/pull/27/commits/38e906ea0e5c44ceb68c3a22f6cd2ca27ef1b447 - let me know if I misconfigured something! 🙇   I verified I get an error on build when trying to import the Label component from the `test-app` with these changes.

I also stole your `'**/*.js',` public entry point line from `headless-form` as I think it makes sense for us too?

Reply:
Perfect! :tada: